proc_CPMG.py

```diff
@@ -18,27 +19,17 @@
     ###{{{loading in data and displaying raw data
     s = find_file(searchstr, exp_type=exp_type,
             expno=nodename, postproc=postproc, lookup=postproc_dict)
- 
-    logging.basicConfig(filename="logfile.txt")
-    stderrLogger=logging.StreamHandler()
-    stderrLogger.setFormatter(logging.Formatter(logging.BASIC_FORMAT))
-    logging.getLogger().addHandler(stderrLogger)
     nEchoes = s.get_prop('acq_params')['nEchoes']
```

Comment:
why is this here -- is a calculation done in the preprocessing that labels the axis correctly?

Reply:
so master also has the `nEchoes` line that I'm referring to.  Also, the current code has not changed this line.

Reply:
yes, this is fixed!